appdev/index.rst

```diff
@@ -6,16 +6,75 @@ Welcome to an open source and free platform under constant scrutiny and improvem
 Introduction
 ------------
 
-.. todo:
+Ubuntu Touch has three types of applications: Web Applications (WebApps), Scopes and Native Applications.
+Applications are packaged, distributed and deployed using a format called `click <https://click.readthedocs.io/en/latest/>`_ packaging.
+Languages of choice are QML or HTML5 for the UI and can be Javascript, Qt, C++, Python, Go for the logic.
+
+Security and app isolation
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+All Ubuntu apps and scopes are confined respecting `AppArmor <https://wiki.ubuntu.com/AppArmor>`_ access control mechanism, meaning they only have access to their own resources and are isolated from other apps and parts of the system. The developer must declare which policy groups are needed for the app or scope to function properly with a `manifest.json` file.
+
+Example `manifest.json` file:
+
+::
+
+    {
+        "policy_groups": [
+            "networking",
+            "webview",
+            "content_exchange"
+        ],
+        "policy_version": 1.3
+    }
+
+Non exhaustive policy groups:
+
```

Comment:
not sure if always available

Reply:
The list looks good to me. It only includes the public policies marked as "common", which is enough to get started with Ubuntu Touch development. Good choice!